lib/taskana-core/src/test/java/acceptance/task/TransferTaskAccTest.java

```diff
@@ -309,21 +309,15 @@ public void testTransferTasksWithListNotSupportingRemove() throws NotAuthorizedE
     public void testTransferTasksWithInvalidTasksIdList() throws NotAuthorizedException, WorkbasketNotFoundException {
         TaskService taskService = taskanaEngine.getTaskService();
         // test with invalid list
-        try {
-            taskService.transferTasks("WBI:100000000000000000000000000000000006", null);
-            Assert.fail("exception was excepted to be thrown");
-        } catch (InvalidArgumentException e) {
-            Assert.assertEquals(e.getMessage(), "TaskIds must not be null.");
-        }
+
+        Throwable t = Assertions.assertThrows(InvalidArgumentException.class, () ->
+            taskService.transferTasks("WBI:100000000000000000000000000000000006", null));
+        Assert.assertEquals(t.getMessage(), "TaskIds must not be null.");
```

Comment:
This is more a question than a comment:
Is it smart to test exception messages? I think this could be a pain if we want to refactor those messages. Maybe a cleanup ticket for the future?

Reply:
good point. we should use a reason enum within the exception and not rely on some strings ... but that's not part of this ticket ;D